### PR TITLE
Fix definition of lower triangular matrix in doc.

### PR DESCRIPTION
**Pull request by RogerFrigola**
Simple fix of copy-paste error in the doc of the Affine bijector.

### CODE REVIEW

File: tensorflow_probability/g3doc/api_docs/python/tfp/bijectors/Affine.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ specified then `scale += IdentityMatrix`. Otherwise specifying a
     `scale_diag` has shape [N1, N2, ...  k], which represents a k x k
     diagonal matrix.
     When `None` no diagonal term is added to `scale`.
-* <b>`scale_tril`</b>: Floating-point `Tensor` representing the diagonal matrix.
+* <b>`scale_tril`</b>: Floating-point `Tensor` representing a lower triangular matrix.
     `scale_diag` has shape [N1, N2, ...  k, k], which represents a k x k
```
Comment (csuter) [inline tensorflow_probability/g3doc/api_docs/python/tfp/bijectors/Affine.md]: Can you fix the mention of scale_diag in the next line, too?

**Comment by RogerFrigola** [comment]
Done.

**Comment by csuter** [comment]
Hey @RogerFrigola, looks like someone internally has identical changes (along with a few other typos/cleanups) in the queue, so I will close this to save further duplication of effort. Thanks very much for proactively sending us the fix, and sorry I didn't realize sooner that someone else was looking at it inside Google. Keep the contributions coming!

**Comment by RogerFrigola** [comment]
No problem. Glad that you are working hard on it internally!

**Pull request closed without merging**